src/js/common/schemaform/save-in-progress/SaveInProgressIntro.jsx

```diff
@@ -76,6 +100,23 @@ export default class SaveInProgressIntro extends React.Component {
           <br/>
         </div>
       );
+    } else if (verifyRequiredPrefill) {
+      alert = (
+        <div>
+          <div className="usa-alert usa-alert-info schemaform-sip-alert">
+            <div className="usa-alert-body">
+              You must be signed in to your account to apply. If your account is verified, your application process can go more smoothly. Here’s why:<br/>
+              <ul>
+                <li>We can submit your Intent to File right away.</li>
```

Comment:
Can we pull this content out too for the same reasons as above?

Reply:
Good idea, moved into `helpers`.